Make sure the order is correct, so that Moar uses own libuv headers.

--- build/mk-moar-pc.in	2023-12-22 05:36:49.000000000 +0800
+++ build/mk-moar-pc.in	2024-01-20 13:45:23.000000000 +0800
@@ -26,7 +26,7 @@
 my $ldopts = '@ldflags@';
 my $ldflags = "-L${libdir} -l${package}";
 my $libs = "${ldflags}";
-my $cflags = "-I${includedir}";
+my $cflags = "-I${includedir}/moar -I${includedir}";
 my $static_libs = '';
 if ('@config@' =~ m/--static/) {
     my $static_libs = "${libdir}/lib${package}.a";
